pkg/driver/node_test.go

```diff
@@ -342,6 +342,20 @@ func TestNodeStageVolume(t *testing.T) {
 				mockMounter.EXPECT().FormatAndMountSensitiveWithFormatOptions(gomock.Eq(devicePath), gomock.Eq(targetPath), gomock.Eq(defaultFsType), gomock.Any(), gomock.Nil(), gomock.Eq([]string{"-N", "13107200"}))
 			},
 		},
+		{
+			name: "success with bigalloc cluster size in ext4",
+			request: &csi.NodeStageVolumeRequest{
+				PublishContext:    map[string]string{DevicePathKey: devicePath},
+				StagingTargetPath: targetPath,
+				VolumeCapability:  stdVolCap,
+				VolumeId:          volumeID,
+				VolumeContext:     map[string]string{BigAllocClusterSizeKey: "16384"},
```

Comment:
Should we add a test case for the unhappy path?

Reply:
Makes sense. Will consider adding unhappy path test case for rest of formatting options (block size, inode size, etc.)